server/complex/src/tasks/ddl.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('lodash');
+
+async function createTasksTables(db) {
+    const tasksExists = await db.schema.hasTable('tasks')
+    if (!tasksExists) {
+        await db.schema.createTable('tasks', function(table) {
+            table.string('title')
+            table.string('description')
+            // Required for conflict resolution
+            table.integer('version')
+            table.increments('id')
+            table.string('status')
+            _.range(25).map(i => table.string(`test${i}`))
```

Comment:
😍 It will be cool to have parameter that will control how many fields we create here.